internal/schema/service.go

```diff
@@ -0,0 +1,93 @@
+package schema
```

Comment:
Instead of a top level schema package, does it make sense to align the package organization with hexagonal architecture?

Reason is that we currently are trying to get rid of all custom packages and follow Kratos/DDD/Hexagonal arch conventions, gist of which is the following

1. the service package is the layer for application services
2. All the domain models and domain services will go in the biz package
3. All repository concerns go in the data package

Some details about the conventions [here](https://go-kratos.dev/docs/intro/layout/). They actually use DDD language so it is easy to connect the dots with DDD.